mesos_slave/tests/test_integration_e2e.py

```diff
@@ -43,5 +42,5 @@ def assert_metrics_covered(aggregator):
         aggregator.assert_metric(v[0])
 
     aggregator.assert_all_metrics_covered()
-
-    aggregator.assert_service_check('mesos_slave.can_connect', status=AgentCheck.OK, count=2)
+    # We should submit 2 service checks per check run because 2 endpoints were tried
+    aggregator.assert_service_check('mesos_slave.can_connect', status=AgentCheck.OK, count=4)
```

Comment:
My guess is integration test is failing cause you have removed the second call to `check.check()`: 
https://github.com/DataDog/integrations-core/pull/4891/files#diff-20651872b7f5656ce3de7545ee50aa39L20
```
 AssertionError: Needed exactly 4 candidates for 'mesos_slave.can_connect', got 2
```